ee/codegen/src/generators/graph-attribute.ts

```diff
@@ -141,27 +141,6 @@ export class GraphAttribute extends AstNode {
         }
       };
 
-      const popSources = (mutableAst: GraphMutableAst): GraphMutableAst => {
```

Comment:
Was just moved to a class method below instead of defined inline each iteration of this while loop. Over time, expect to do the same for the rest of these methods